src/utils/Validator.js

```diff
@@ -0,0 +1,40 @@
+import { NAME_DELIMITER, ERROR_MESSAGE, VALIDATOR } from '../constants/index.js';
+
+const Validator = {
+  checkBlank(input) {
+    if (input.includes(' ')) {
+      throw new Error(ERROR_MESSAGE.blank);
+    }
+  },
+  checkLastComma(input) {
+    if (input[input.length - 1] === NAME_DELIMITER) {
+      throw new Error(ERROR_MESSAGE.lastComma);
+    }
+  },
+  checkNameLength(input) {
+    if (input.split(NAME_DELIMITER).some((name) => name.length === 0)) {
+      throw new Error(ERROR_MESSAGE.nameLength);
+    }
+  },
+  checkDuplicate(input) {
+    const value = input.split(NAME_DELIMITER);
+    if (new Set(value).size !== value.length) {
+      throw new Error(ERROR_MESSAGE.duplicate);
+    }
+  },
+  checkNames(input) {
+    this.checkBlank(input);
+    this.checkLastComma(input);
+    this.checkNameLength(input);
+    this.checkDuplicate(input);
+  },
+  checkIntegerNumber(input) {
+    input.split('').forEach((char) => {
+      if (!VALIDATOR.integerString.includes(char)) {
+        throw new Error(ERROR_MESSAGE.notInteger);
+      }
+    });
+  },
+};
```

Comment:
객체로 다 감싼 이유가 있을까요.,.,?

Reply:
이 부분은 깊게 고민하지 않았는데, Validator.js 파일이 이미 메서드를 묶어주고 있어서 감싸지 않아도 좋을 것 같습니다!
```suggestion
export checkBlank(input) {
  if (input.includes(' ')) {
    throw new Error(ERROR_MESSAGE.blank);
  }
}

export checkLastComma(input) {
...
```